LondonBritishLibrary/orient/BLorient4849.xml

```diff
@@ -0,0 +1,608 @@
+                            </msItem>
+                            <msItem xml:id="ms_i4.6">
+                                <locus from="121v" to="122r"/>
+                                <title ref="LIT4035SenkessarDL#Pag3SalMalkaSedeq'"/>
+                                <incipit xml:lang="gez"><add resp="PRS8999Strelcyn">ሰላም፡</add> ለመልከ፡ ጼዴቅ፡ አምሳሉ፡ ወሱታፉ። </incipit>
+                            </msItem>
+                            <msItem xml:id="ms_i4.7">
+                                <locus from="122r" to="122v"/>
+                                <title ref="LIT4035SenkessarDL#Pag3Sarapyon"/>
+                                <incipit xml:lang="gez">ወበዛቲ፡ ዕለተ፡ ካዕበ፡ አዕረፈ፡ ሰራጵዮን፡ ዘለንዱን፡ ወዝንቱ፡ ቅዱስ፡ ተምህረ፡ መጻሕፍተ፡ ቤተ፡ ክርስቲያን፡ ወትርጓሜሁ፡ ወመነነ፡ 
+                                    ኵሎ፡ ዓለመ፡ ወንዋየ፡ ዝንቱ፡ ዓለም።</incipit>
+                            </msItem>
+                            <msItem xml:id="ms_i4.8">
+                                <locus target="122v"/>
+                                <title ref="LIT4035SenkessarDL#Pag3SalSarapyon"/>
+                                <incipit xml:lang="gez">ሰላም፡ እብል፡ ለሰራጵዮን፡ ክቡር።</incipit>
+                            </msItem>
+                        </msItem>
+                    </msContents>
+                    <physDesc>
+                        <objectDesc form="Codex">
+                            <supportDesc>
+                                <support>
+                                    <material key="parchment"/>
+                                </support>
+                                <extent>
+                                    <measure unit="leaf">1+122+2</measure>
+                                    <measure unit="page" type="blank">1</measure><locus target="#8r"/>
+                                    <dimensions type="outer" unit="mm">
+                                        <height>165</height>
+                                        <width>95</width>
+                                    </dimensions>
+                                </extent>
+                                <collation>Paper endleaves added when the codex was repaired in the 
+                                    <placeName ref="INS00001BL">British Museum</placeName></collation>
+                            </supportDesc>
+                            <layoutDesc>
+                                <layout writtenLines="21"/>                                
+                            </layoutDesc>   
+                        </objectDesc> 
+                        <handDesc>
+                            <handNote script="Ethiopic" xml:id="h1">
+                                <desc>Written in a good hand.</desc>
+                                <date notBefore="1600" notAfter="1799" resp="PRS8999Strelcyn"/>
+                            </handNote>
+                            <handNote xml:id="h2" script="Ethiopic">
```

Comment:
I think it is a composite MS with two MSS of Dersana Mikael, one partial/incomplete, one full, or possibly this particular homily for Maggabit is not in the main part so was added as msItem 1, see https://github.com/BetaMasaheft/Documentation/issues/2563

Reply:
I see, that is better to define ms_i1 as an individual text part, even though there is no indication in the Strelcyns catalogue of any change of hands or any changes in layout and dimensions. However, I am still wondering what to do with the twenty pages filled with miniatures in the beginning? They can be defined as part of part 1 (= ms_i2) or part 2 (= ms_i2 to ms_i5), they can form a text part on their own or they can be seen as later addition, that should be included into the physDesc of the record concerning the manuscript as a whole. I think, I would opt for creating a third text part for the miniatures in the decoDesc.

Reply:
I have just mentioned that ms_i1 ends at folio 32r, while ms_i2 begins at folio 32v.  I think it is rather unlikely that these are separate codicological units. In that case, either folio 32r or folio 32v must have been added later, after the two parts had been bound together. Whatever the reason for putting ms_i1 first, I think that each ms_item was written one after the other.